requirements.txt

```diff
@@ -3,40 +3,25 @@
 # process, which may cause wedges in the gate later.
 
 pbr>=0.11,<2.0
-#
+
 # The greenlet package must be compiled with gcc and needs
 # the Python.h headers. Make sure you install the python-dev
 # package to get the right headers...
 greenlet>=0.3.2
 
-# < 0.8.0/0.8 does not work, see https://bugs.launchpad.net/bugs/1153983
-SQLAlchemy>=0.9.7,<=0.9.99
-anyjson>=0.3.3
 eventlet>=0.17.3
 PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
-sqlalchemy-migrate>=0.9.5
 httplib2>=0.7.5
```

Comment:
It's currently required for keystone auth in searchlight/common/auth.py, but that custom code should be ripped out